src/datachain/remote/studio.py

```diff
@@ -230,6 +222,46 @@ def ls(self, paths: Iterable[str]) -> Iterator[tuple[str, Response[LsData]]]:
     def ls_datasets(self) -> Response[LsData]:
         return self._send_request("datachain/ls-datasets", {})
 
+    def edit_dataset(
+        self,
+        name: str,
+        new_name: Optional[str] = None,
+        description: Optional[str] = None,
+        labels: Optional[list[str]] = None,
+    ) -> Response[DatasetInfoData]:
+        body = {
+            "dataset_name": name,
+        }
+
+        if new_name is not None:
+            body["new_name"] = new_name
+
+        if description is not None:
+            body["description"] = description
+
+        if labels is not None:
+            body["labels"] = labels  # type: ignore[assignment]
+
+        return self._send_request(
+            "datachain/edit-dataset",
```

Comment:
related to the groups of commands, etc - let's please make API also clean

`datachain/datasets/ls` 
`datachain/datasets/rename` 

-etc 

`rm-dataset` and similar look very weird for REST api.

Reply:
It could even be a single endpoint that lists on `GET`, renames via `PUT` (assuming dataset name is not part of a resource URI as `PUT` has to be idempotent), removes via `DELETE`.

Since it's a part of a team, ideally the URI should be `/datachain/<team>/datasets`.

Reply:
Added this comment to the #576 as well. We can take it as a part of refactor.